.github/workflows/Kogito-Operator-Unit-tests.yaml

```diff
@@ -29,26 +31,36 @@ jobs:
             ${{ runner.os }}-go-cache-
       - name: Install cover
         run: go get golang.org/x/tools/cmd/cover
-      - run: go mod tidy
       - name: Validate codcov yaml file
         run: curl -vvv --data-binary @codecov.yml https://codecov.io/validate
-      - name: Test cmd dir
+      - name: Cache testenv
+        uses: actions/cache@v2
+        with:
+          path: $(pwd)/${ENVTEST_ASSETS_DIR}
+          key: ${{ runner.os }}-testbin-${{ env.OPERATOR_SDK_VERSION }}
+          restore-keys: |
+              ${{ runner.os }}-testbin-${{ env.OPERATOR_SDK_VERSION }}
+      - name: Run unit tests
         run: |
-          go test ./cmd/... -count=1 -p=1 -coverprofile=cmd-coverage.txt -covermode=atomic
+          ./hack/go-test.sh
```

Comment:
Because we are running `lint` and `fmt` as part of our `make test`. Since we already have a different GH action to check those I would just run the unit tests here. :)

Reply:
ok. as we have a Makefile that will call different scripts, I am just worried about bypassing the Makefile and maybe later changes in the `test` command that would not be taken into account the GHAction

Reply:
Yeah, that is true, although we had this same structure earlier as well, since in github actions we only want to run unit tests but in `make test` we do other things like check lint and format the code as well.